examples/invRegex.py

```diff
@@ -28,56 +28,50 @@ def __repr__(self):
         return '['+self.charset+']'
     def makeGenerator(self):
         def genChars():
-            for s in self.charset:
-                yield s
+            yield from self.charset
         return genChars
 
-class OptionalEmitter(object):
+class OptionalEmitter:
     def __init__(self,expr):
         self.expr = expr
     def makeGenerator(self):
         def optionalGen():
             yield ""
-            for s in self.expr.makeGenerator()():
-                yield s
+            yield from self.expr.makeGenerator()()
         return optionalGen
 
-class DotEmitter(object):
+class DotEmitter:
     def makeGenerator(self):
         def dotGen():
-            for c in printables:
-                yield c
+            yield from printables
         return dotGen
 
-class GroupEmitter(object):
+class GroupEmitter:
     def __init__(self,exprs):
         self.exprs = ParseResults(exprs)
     def makeGenerator(self):
         def groupGen():
             def recurseList(elist):
                 if len(elist)==1:
-                    for s in elist[0].makeGenerator()():
-                        yield s
+                    yield from elist[0].makeGenerator()()
                 else:
                     for s in elist[0].makeGenerator()():
                         for s2 in recurseList(elist[1:]):
                             yield s + s2
             if self.exprs:
-                for s in recurseList(self.exprs):
-                    yield s
+                yield from recurseList(self.exprs)
         return groupGen
 
-class AlternativeEmitter(object):
+class AlternativeEmitter:
     def __init__(self,exprs):
         self.exprs = exprs
     def makeGenerator(self):
         def altGen():
             for e in self.exprs:
-                for s in e.makeGenerator()():
-                    yield s
+                yield from e.makeGenerator()()
```

Comment:
I always wanted this in py2.  If this change gets accepted, then I guess all my other arguments are mute.